lua/auto-session/init.lua

```diff
@@ -882,6 +908,12 @@ function SetupAutocmds()
     { complete = AutoSession.CompleteSessions, bang = true, nargs = "*", desc = "Delete Session" }
   )
 
+  vim.api.nvim_create_user_command(
+    "SessionPurgeOrphaned",
+    SessionPurgeOrphaned,
+    { bang = true, desc = "Remove all orphaned sessions with no directory left" }
```

Comment:
What `bang = true` does. Do we need it here?

Reply:
Unless you're handling `bang` behaviour differently (which you aren't) you shouldn't need bang. This is only useful if you'd like to support something like `SessionPurgeOrphaned!`.